app/scripts/views/settings/devices.js

```diff
@@ -20,6 +24,29 @@ define(function (require, exports, module) {
   var FIREFOX_DOWNLOAD_LINK = 'https://www.mozilla.org/firefox/new/?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-devices';
   var FORCE_DEVICE_LIST_VIEW = 'forceDeviceList';
 
+  function formatConnectedTimestamp (timestamp) {
+    return $.timeago(timestamp, {
+      settings: {
+        strings: {
+          day: t('a day'),
+          days: t('%d days'),
+          hour: t('about an hour'),
+          hours: t('about %d hours'),
+          inPast: t('any moment now'),
+          minute: t('about a minute'),
+          minutes: t('%d minutes'),
+          month: t('about a month'),
+          months: t('%d months'),
+          seconds: t('less than a minute'),
```

Comment:
This one feels a bit cumbersome to me. Would something like `just now` be better?